### PR TITLE
chore(changelog generation): Improve formatting

### PR DESCRIPTION
**Pull request by krancour**

### CODE REVIEW

File: changelog/changelog.go [hunk 1]
```diff
@@ -5,28 +5,28 @@ import (
 )
 
 const (
-	tplStr = `{{.OldRelease}} -> {{.NewRelease}}
+	tplStr = `### {{.OldRelease}} -> {{.NewRelease}}
 
 {{ if (len .Features) gt 0 }}
-# Features
+##### Features
```
Comment (vdice) [inline changelog/changelog.go]: looks like `Features` gets a slightly smaller header size than the rest (one more `#`) -- intentional?

**Comment by vdice** [comment]
Thoughts on adding assertions around this formatting in existing [template test](https://github.com/deis/deisrel/blob/master/changelog/changelog_test.go)?  Not sure I have strong opinion either way, but it would probably be a quick addition.  Otherwise, LGTM.

**Comment by krancour** [comment]
@vdice I've updated the "missing" strings in `changelog_test.go`

File: changelog/changelog.go [hunk 2]
```diff
@@ -5,28 +5,28 @@ import (
 )
 
 const (
-	tplStr = `{{.OldRelease}} -> {{.NewRelease}}
+	tplStr = `### {{.OldRelease}} -> {{.NewRelease}}
```
Comment (arschles) [inline changelog/changelog.go]: maybe this should be `##`, since it's the overall header for the release

Comment (arschles) [inline changelog/changelog.go]: err, nevermind. I see the below ones are `####`

Comment (krancour) [inline changelog/changelog.go]: The levels of heading were selected to remain uniform with our existing changelogs.

**Pull request merged**